app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
Yeah, I think long-term we're looking at subclassing MultiValueInput and replacing `build_field`